### DISCUSSION TITLE
{ author: user } make this without pinging the guy who reacted please!

### DISCUSSION THREAD

**Discussion by UnDeadLolomat** [eritislami/evobot#685] [General]
Pinging me or others every time is kinda annoying ngl. Please make the thing to like write the author big, but not ping them.
Like that : **Lolomat** paused the Music! and not @Lolomat paused the music!

**Comment by Takiyo0** [comment]
you can add `"allowedMentions": { "users": [] }` in `message.channel.send` option. It'll tag the user but not pinging the user
```
message.channel.send(`content that tag the user`, { "allowedMentions": { "users": [] } })
```
and will result 
![image](https://user-images.githubusercontent.com/60313514/111998174-5681f700-8b4e-11eb-9e7e-6885d4827005.png)

but if you just want to tell the username or tag, you can use
```
message.channel.send(`**${message.author.tag}** your content here`)
```
and will result
![image](https://user-images.githubusercontent.com/60313514/111998617-c85a4080-8b4e-11eb-87b4-556c6f008150.png)


hope this work

**Reply by UnDeadLolomat** [reply]
That worked perfectly fine, thank u ! I used { author: user }), { "allowedMentions": { "users": [] }} !
